### PR TITLE
Updated hello-world to the new style of exercise

### PR DESCRIPTION
**Pull request by Nik-Mouz**
Removed string manipulation from hello-world exercise to be inline with the new style of hello-world exercise across the site as discussed in #110
Edited model answer, tests and unimplemented answer

### CODE REVIEW

**Comment by stkent** [comment]
I'll pick this up once #110 is resolved 👍

**Comment by stkent** [comment]
I will be able to get to this on Sunday!

File: exercises/hello-world/TUTORIAL.md [hunk 1]
```diff
@@ -3,11 +3,7 @@ https://github.com/exercism/kotlin/blob/master/exercises/hello-world/TUTORIAL.md
```
Comment (stkent) [inline exercises/hello-world/TUTORIAL.md]: Please add a `hints.md` file to direct users to this tutorial (as in https://github.com/exercism/java/blob/master/exercises/hello-world/.meta/hints.md).

Comment (stkent) [reply exercises/hello-world/TUTORIAL.md]: Please also port the new Introduction and Exercise Structure sections from the Java tutorial file.

**Comment by stkent** [comment]
Great, thanks @Nik-Mouz! LGTM 🚢

**Comment by katrinleinweber** [comment]
Was it intended to remove all the info about un-`@Ignore`-ing tests here?

**Comment by stkent** [comment]
Yes, though the intent was to bring that information back in a similar place to that used by the Java track now, but that second part never happened.

**Pull request merged**